api/version/version.go

```diff
@@ -26,7 +26,7 @@ import (
 var (
 	// MinClusterVersion is the min cluster version this etcd binary is compatible with.
 	MinClusterVersion = "3.0.0"
-	Version           = "3.5.0-alpha.0"
+	Version           = "3.5.0"
```

Comment:
I think main should be 3.6 or 3.5.0+git?

Reply:
Good point! Fine with either, what was done in the past?

Reply:
`3.5.0-pre -> 3.5.0-pre -> 3.5.0-alpha.0`

So we should call it `3.6.0-pre`, 
unless you want to follow the sorting recommendation from the other comment: `v3.6.0-adev`

Reply:
Thanks, done!